Makefile

```diff
@@ -4,7 +4,7 @@ TARGETS      := darwin/amd64 darwin/arm64 linux/amd64 linux/arm64 windows/amd64
 BINNAME      ?= osm
 DIST_DIRS    := find * -type d -exec
 CTR_REGISTRY ?= openservicemesh
-CTR_TAG      ?= latest
+CTR_TAG      ?= latest-main
```

Comment:
seems like rest of the codebase uses this value: https://github.com/openservicemesh/osm/blob/5f54056bdf6238609e26cf182ac091f43deba60d/charts/osm/values.yaml#L15

Makes it easier to get started since you don't need to override this.